### DISCUSSION TITLE
Table column filter: infinite scroll or search-only options

### DISCUSSION THREAD

**Discussion by krzd7** [grafana/grafana#38225] [Dashboards, Panels & Visualizations]
**What would you like to be added**: I believe it is not possible right now, but I have a table with several rows and there are columns that I'd love being able to filter/search. However, I have noticed that it degrades performance a lot, mainly because it loads all filter options. It would be interesting having a "search only" option (no filter items checkboxes) or having an infinite scroll added to the filter items list.

**Why is this needed**: usability and performance improvements. I'm not sure, but depending on the amount of data it could lead the browser to a crash or at least become unresponsive for a while.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
